packages/kbn-scout-reporting/src/reporting/report/persistence/mappings.ts

```diff
@@ -0,0 +1,198 @@
+/*
```

Comment:
Is there a reason we don't have a mapping for the `host` properties in here?

Reply:
The `host` and `event` properties are covered by ECS mappings.